api/endpoints.go

```diff
@@ -533,7 +534,16 @@ func (a *api) handlePUTSystemDir(c jape.Context) {
 }
 
 func (a *api) handleGETTPoolFee(c jape.Context) {
-	c.Encode(a.tpool.RecommendedFee())
+	var responseFormat string
```

Comment:
This should be one line `a.writeResponse(c, http.StatusOK, TPoolResp(a.tpool.RecommendedFee()))`

Reply:
`Cannot use t (variable of type TPoolResp) as float64 value in struct literal` when changing `TPoolResp` to `types.Currency` along with this change.

Reply:
the non prometheus tpool response is I think the only response that's not json formatted `"30000000000000000000"`

Reply:
```go
// PrometheusMetric returns Prometheus samples for the hosts tpool fee.
func (t TPoolResp) PrometheusMetric() (metrics []prometheus.Metric) {
	return []prometheus.Metric{
		{
			Name:  "hostd_tpool_fee",
			Value: types.Currency(t).Siacoins(),
		},
	}
}
```